peer.go

```diff
@@ -2232,6 +2240,13 @@ func (p *peer) handleLinkFailure(failure linkFailureReport) {
 		}
 	}
 
+	// If this is a permanent failure, we will mark the channel borked.
+	if failure.linkErr.PermanentFailure && lnChan != nil {
+		if err := lnChan.State().MarkBorked(); err != nil {
+			peerLog.Errorf("Unable to mark channel borked: %v", err)
```

Comment:
nit-est nit, but maybe log the channel ID as well?

Reply:
Done!